client/lib/upgrades/actions/cart.js

```diff
@@ -60,13 +61,16 @@ function addItems( items ) {
 		shouldBundleWithPremium = shouldBundleWithPremium && ! isPlan( selectedSite.plan ) && ! cartItems.hasPlan( cart );
 	}
 
-	if ( shouldBundleWithPremium ) {
+	//add premium plan for users not on the personalPlan group.
+	if ( shouldBundleWithPremium && abtest( 'personalPlan' ) === 'hide' ) {
 		items = [ cartItems.premiumPlan( 'value_bundle', { isFreeTrial: false } ) ].concat( items );
 	}
+
 	const extendedItems = items.map( ( item ) => {
 		const extra = assign( {}, item.extra, {
 			context: 'calypstore',
-			withPlansOnly: domainsWithPlansOnlyTestEnabled && ! freeTrialsEnabled ? 'yes' : ''
+			withPlansOnly: domainsWithPlansOnlyTestEnabled && ! freeTrialsEnabled ? 'yes' : '',
+			withPersonalPlan: personalPlanTestEnabled ? 'yes' : ''
 		} );
 
 		return assign( {}, item, { extra } );
```

Comment:
I realize this was here before, but if we are already using `assign` from **lodash** then we might as well use `merge` to save some steps

```js
const extendedItems = items.map( item => merge( item, {
	extra: {
		context: 'calypstore',
		withPlansOnly: …
		withPersonalPlan: …
	}
} ) );
```

`merge()` is a recursive `assign()` that will only wipe out keys if they are passed in as `undefined`